style.css

```diff
@@ -22,18 +22,139 @@ body {
   margin-right: 24px;
 }
 
-.logo {
+/* .logo {
   margin-right: auto;
   margin-left: 24px;
+} */
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  background-color: #fff;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
 }
 
-.desk {
+.logo {
+  font-size: 1.5rem;
+  font-weight: bold;
+}
+
+.menu-toggle {
   display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px 0;
+  background-color: #396df2;
+  transition: transform 0.2s ease-in-out;
+}
+
+nav {
+  display: flex;
+}
+
+nav ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+  display: inline;
+  left: 2.4%;
+  right: 88.27%;
+  top: 77.81%;
+  bottom: 18.88%;
+  margin-left: 15px;
+}
+
+nav li {
+  list-style: none;
+  display: inline;
+  left: 2.4%;
+  right: 88.27%;
+  top: 77.81%;
+  bottom: 18.88%;
+  margin-left: 15px;
+  margin-right: 1rem;
+}
+
+nav a {
+  text-decoration: none;
+  color: #333;
+  font-weight: bold;
+}
+
+.close-button {
+  position: absolute;
+  top: -6.4%;
+  right: 8rem;
+  font-size: 2.8rem;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  margin-left: 81.9%;
+  color: #396df2;
+}
+
+@media screen and (max-width: 768px) {
+  .menu-toggle {
+    display: block;
+  }
+
+  nav {
+    position: absolute;
+    top: 3rem;
+    left: 0;
+    width: 100%;
+    height: 97vh;
+    background-color: #0339c2e7;
+    padding: 1rem;
+    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
+    transform: translateX(-100%);
+    transition: transform 0.2s ease-in-out;
+  }
+
+  nav a {
+    text-decoration: none;
+    color: #fff;
+    font-weight: bold;
+    font-size: 32px;
+  }
+
+  nav.show {
+    transform: translateX(0);
+  }
+
+  nav ul {
+    flex-direction: column;
+    align-items: center;
+  }
+
```

Comment:
- [ ] Nice work working on the Mobile menu I can see the functionality is awesome 👏 , however, we need to stick to the styling requirements and mock it strictly.  Kindly make sure your design matches the template by using the same font-family, layout, background..... check the difference below

Your Design | The template 
--|---
<img width="420" alt="image" src="https://user-images.githubusercontent.com/65068771/232979425-ef236288-0320-4b67-95b1-0fc00e46e7d1.png"> | <img width="420" alt="image" src="https://user-images.githubusercontent.com/65068771/232979565-9fde1945-b2ed-4549-bdbe-fa0c1da7902e.png">